redfish-core/lib/service_root.hpp

```diff
@@ -23,6 +23,40 @@
 namespace redfish
 {
 
+
+inline void
+    handleACFWindowActive(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
+{
+    crow::connections::systemBus->async_method_call(
+        [asyncResp](const boost::system::error_code ec,
+                    const std::variant<bool>& retVal) {
+            if (ec)
+            {
+                BMCWEB_LOG_ERROR
+                    << "Failed to read ACFWindowActive property";
+                // Default value when panel app is unreachable.
+                asyncResp->res.jsonValue["Oem"]["IBM"]["ACFWindowActive"] =
+                    false;
+            }
+            const bool* isACFWindowActive = std::get_if<bool>(&retVal);
+            if (isACFWindowActive == nullptr)
+            {
+                BMCWEB_LOG_ERROR << "nullptr for ACFWindowActive";
+                messages::internalError(asyncResp->res);
+            }
```

Comment:
return here and get rid of the else?

Reply:
Yes, I'll return here to follow the coding style.